planning/autoware_mission_planner_universe/src/lanelet2_plugins/default_planner.cpp

```diff
@@ -207,6 +207,13 @@ bool DefaultPlanner::check_goal_footprint_inside_lanes(
     boost::geometry::correct(poly);
     ego_lanes.push_back(poly);
   }
+  // If the goal is on the very beginning of the closest_lanelet_to_goal, baselink ~ rear part of
+  // ego footprint is outside of it. To tolerate it, add previous lanelet
+  for (const auto & ll : route_handler_.getPreviousLanelets(closest_lanelet_to_goal)) {
+    boost::geometry::convert(ll.polygon2d().basicPolygon(), poly);
+    boost::geometry::correct(poly);
+    ego_lanes.push_back(poly);
+  }
```

Comment:
memo for limitation:
If the previous is very short, it may stick out. In such a case, it is necessary to take all previous lanes for footprints.
This is not the case here.